index.html

```diff
@@ -37,8 +38,199 @@ <h1>I'm Joseph Glad to meet you!</h1>
+            <li><a href="">Html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+         </ul>
+
+         <button class="button"><a href="">See Project</a></button>
+        
+    </section>
+    <section class="grid-item">
+        <img class="snap" src="Images/Snapshoot Portfolio (3).png" alt="">
+        <h2 class="multi">Multi-Post
+            Stories</h2>
+        <div class="container">
+            <div class="canopy">
+                <p>CANOPY</p>
+            </div>
+            
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>Backend Dev</p>
+            </div>
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>2015</p>
+            </div>
+        </div>
+        <div class="content">
+            <p>
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required.
+            </p>
+        </div>
+        
+        <ul class="link">
+            <li><a href="">Html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+         </ul>
 
+         <button class="button"><a href="">See Project</a></button>
+        
+    </section>
 
+     </section>
+    <section class="about">
+        <h2 class="multi">About Myself</h2> 
+    <p>
+        I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+    </p>
+        <p class="contain">LET’S CONNECT</p>
     
+    <ul class="about-social"> 
+        <li> <a href=""><img src="Images/github.svg" alt=""></a></li>
+        <li> <a href=""><img src="Images/Linkedin icon.svg" alt=""></a></li>
+        <li> <a href=""><img src="Images/Vector.svg" alt=""></a></li>
+        <li> <a href=""><img src="Images/twitter.svg" alt=""></a></li>
+        <li> <a href=""><img src="Images/m vector.svg" alt=""></a></li>
+        
+        
+    </ul>
+
+    <button class="about-button"><a href="">Get my resume</a></button>
+
+
+<div class="lang">
+    <h3>Languages</h3>
+    <img src="Images/arrow-down-sign-to-navigate.png" alt="">
+</div>
+
+<ul class=" basic"> 
+    <li><img src="Images/Ellipse 1.png" alt=""> <p>Javascript</p> </li>
+    <li><img src="Images/Ellipse 1 (1).png" alt=""> <p>HTML</p> </li>
+    <li><img src="Images/Ellipse 1 (2).png" alt=""> <p>CSS</p></li>
+</ul>
+
+    <ul class="lang-skills">
+        <li> <p>Frameworks</p> <img src="Images/right-arrow.png" alt=""></li>
+        <li class="skills"> <p>Skills</p> <img src="Images/right-arrow.png" alt=""></li>
+    </ul>
+    <div><img class="shape" src="Images/Separator Bottom.png" alt=""></div>
+     <div><img class="shape-1" src="Images/Separator Bottom.png" alt=""></div> 
```

Comment:
**OPTIONAL** - This section of your page does not look exactly like the Figma design template. Please fix this 🙏.
Your project:
<img width="378" alt="wrawrgawg" src="https://user-images.githubusercontent.com/73607512/219880745-b14d0fdc-f8ed-447f-8a4a-b3e4c4562229.PNG">

Design template:
<img width="146" alt="ethethe" src="https://user-images.githubusercontent.com/73607512/219880788-67d4e16b-571e-4dc9-93d6-d5dafd363c10.PNG">